nservicebus/messaging/discard-old-messages.md

```diff
@@ -40,8 +40,9 @@ TimeToBeReceived relies on underlying functionality in the transport infrastruct
 
 ### MSMQ transport
 
-MSMQ continuously checks the TimeToBeReceived attribute of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space reclaimed. MSMQ will however only allow a single TimeToBeReceived for all messages in a transaction. It will silently copy the TimeToBeReceived from the first message enlisted to all other messages in the transaction, leading to a potential message loss scenario. This issue, unfortunately, make it impossible to support setting TimeToBeReceived for messages sent from a transactional MSMQ endpoint.
+MSMQ continuously checks the TimeToBeReceived attribute of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space reclaimed. MSMQ will however only allow a single TimeToBeReceived for all messages in a transaction. It will silently copy the TimeToBeReceived from the first message enlisted to all other messages in the transaction, leading to a potential message loss scenario.
 
+If TTBR is required than [immediate message dispatch](send-a-message.md#dispatching-a-message-immediately) can be used with the caveat that this message will be send even though an error occurs after sending.
```

Comment:
So, if we use immediate dispatch, we can the TTBR attribute can be used? 

>If TTBR is required than [immediate message dispatch](send-a-message.md#dispatching-a-message-immediately) can be used with the caveat that this message will be send even though an error occurs after sending.

Minor Typo.  Also I believe the warning needs to be much more explicit. For example:

If TTBR is required then, [immediate message dispatch](send-a-message.md#dispatching-a-message-immediately) can be used. However, when using immediate dispatch, messages will get sent or published, even if the message handling resulted in an error. Therefore, based on the number of retries that are configured on the endpoint, the same message could be sent or published multiple times. This behavior needs to be taken into consideration when choosing to use immediate dispatch.